nucleus/parent/pom.xml

```diff
@@ -1074,7 +1074,7 @@
                         <dateFormatTimeZone>UTC</dateFormatTimeZone>
                         <commitIdGenerationMode>full</commitIdGenerationMode>
                         <includeOnlyProperties>
-                            <includeOnlyProperty>git.branch</includeOnlyProperty>
+                            <!--includeOnlyProperty>git.branch</includeOnlyProperty--><!-- not reproducible -->
```

Comment:
If this is not set - I suppose the https://github.com/eclipse-ee4j/glassfish/blob/f985ae7390941832b0f2f7c6a2ae651723598561/appserver/distributions/glassfish-common/src/main/resources/glassfish/config/branding/glassfish-version.properties#L21 and its usages should be updated.